TodoApp/src/TodoApp.Services/Todos/UpdateTodoService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using TodoApp.Contracts;
+using TodoApp.Contracts.Helpers;
+using TodoApp.Contracts.Models;
+using TodoApp.Contracts.Repositories;
+using TodoApp.Contracts.Services.Todos;
+
+namespace TodoApp.Services.Todos
+{
+    class UpdateTodoService: IUpdateTodoService
```

Comment:
![image](https://user-images.githubusercontent.com/15132110/32231472-fe592c6a-be55-11e7-8b8c-29fd6e680837.png)
So where exactly is the problem with explicit modifiers? :stuck_out_tongue: